pkg/kotscli/kotscli.go

```diff
@@ -89,7 +91,8 @@ type AirgapUpdateOptions struct {
 }
 
 func AirgapUpdate(opts AirgapUpdateOptions) error {
-	kotsBinPath, err := goods.MaterializeInternalBinary("kubectl-kots")
+	materializer := goods.NewMaterializer(defaults.EmbeddedClusterHomeDirectory())
```

Comment:
shouldn't this use the runtimeconfig?

Reply:
this pr only attempts to consolidate the directories. not allow them to be overwritten. i will do that in a follow up